compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -815,6 +817,48 @@ void OperationExporter::visit(luci::CircleBatchMatMul *node)
                 CreateBatchMatMulOptions(_ctx.builder, node->adj_x(), node->adj_y()).Union());
 }
 
+void OperationExporter::visit(luci::CircleBidirectionalSequenceLSTM *node)
+{
+  auto bidi_lstm_outs = loco::succs(node);
+  assert(int32_t(bidi_lstm_outs.size()) == 2);
```

Comment:
I'm wondering this is always 2.
```cxx
bool visit(luci::CircleBidirectionalSequenceLSTM *node) final
  {
    if (node->merge_outputs())
    {
      store_outputs(node, 1);
    }
    else
    {
      store_outputs(node, 2);
    }
    return true;
  }
```
Because, as this codes, it could be 1.

Reply:
> I'm wondering this is always 2.
> 
> ```c++
> bool visit(luci::CircleBidirectionalSequenceLSTM *node) final
>   {
>     if (node->merge_outputs())
>     {
>       store_outputs(node, 1);
>     }
>     else
>     {
>       store_outputs(node, 2);
>     }
>     return true;
>   }
> ```
> 
> Because, as this codes, it could be 1.

You are right, for the currently available recipes, the assert would pass. In general, it should have a range check between 1 to 2. I will reflect this in the subsequent commit. Thanks.

Reply:
Got it:) then `format-checker` should be solved to merge this PR.